src/libs/OptionsListUtils.ts

```diff
@@ -1428,6 +1431,13 @@ function getOptions(
     orderedReports.reverse();
 
     const allReportOptions: ReportUtils.OptionData[] = [];
+    const isTaskActionTypeForParticipants = actionTypeForParticipants === CONST.REPORT.TYPE.TASK;
+    const isMoneyRequestActionTypeForParticipants =
```

Comment:
If `isTaskActionTypeForParticipants=true`, `isMoneyRequestActionTypeForParticipants` will be false, we don't have to check it in this case
```js
const isMoneyRequestActionTypeForParticipants = !isTaskActionTypeForParticipants && (
        actionTypeForParticipants === CONST.IOU.REQUEST_TYPE.MANUAL ||
        actionTypeForParticipants === CONST.IOU.REQUEST_TYPE.SCAN ||
        actionTypeForParticipants === CONST.IOU.REQUEST_TYPE.DISTANCE ||
        actionTypeForParticipants === CONST.IOU.TYPE.SPLIT);